src/components/Console/ConsoleCard.jsx

```diff
@@ -12,15 +12,15 @@ import './ConsoleCard.scss';
 	Props:
 	logs -  'array or string', required. Representing the logs to be displayed.
 	header - 'component', optional. The header to be displayed at the CardHeader
-
 	cardBodyClass - 'string', optional. Additional CSS classes for the CardBody element.
+	isWrap - 'bool', optional. Initially makes text wrap
 */
 
-export function ConsoleCard ({ logs, header, cardBodyClass = '' }) {
+export function ConsoleCard ({ logs, header, cardBodyClass = '', isWrap = false }) {
```

Comment:
we should come up with better naming here, something like `shouldWrap` or `wrapText` ...